modules/common/data/gem_config.pb.txt

```diff
@@ -1,4 +1,5 @@
 vehicle_param {
+  brand: GEM
```

Comment:
Would it be possible to move the config to `modules/calibration/data/<vehicle_id>/` as it looks like per-vehicle configuration instead of global common data?